### PR TITLE
Fix SQL validation for imports

### PR DESCRIPTION
**Pull request by pandah3**
## Description

With the changes to SQL validation in #640, we need to identify if the SQL validation is running via an import or as a standalone command.

## Steps to Test=

1. Check out PR.
1. Run `npm run build`
1. Run `node ./dist/bin/vip-import-sql @103.staging /Users/joannelee/Downloads/test.sql --search-replace="hello.com,goodbye.com"`
1. This mainly works in conjuction with #639, where we're changing the output logs for imports

### CODE REVIEW

File: src/lib/validations/sql.js [hunk 1]
```diff
@@ -247,24 +247,29 @@ const perLineValidations = ( line: string ) => {
 	lineNum += 1;
 };
 
+const execute = ( line: string, isImport: boolean = true ) => {
+	perLineValidations( line, isImport );
+};
+
+const postLineExecutionProcessing = async ( { fileName, isImport }: PostLineExecutionProcessingParams ) => {
```
Comment (jblz) [inline src/lib/validations/sql.js]: Do we need this function any more? It's a 1:1 proxy with `postValidation` as far as I can see.

File: src/lib/validations/sql.js [hunk 2]
```diff
@@ -247,24 +247,29 @@ const perLineValidations = ( line: string ) => {
...
+
 export const staticSqlValidations = {
-	execute: ( line: string ) => {
-		perLineValidations( line );
-	},
-	postLineExecutionProcessing: async ( { fileName, isImport }: PostLineExecutionProcessingParams ) => {
-		await postValidation( fileName, isImport );
-	},
+	execute,
+	postLineExecutionProcessing,
```
Comment (jblz) [inline src/lib/validations/sql.js]: same as above, can we just change this to:
```suggestion
	postLineExecutionProcessing: postValidation,
```

...?

File: src/lib/validations/sql.js (hunk 1 above)
Comment (pandah3) [reply src/lib/validations/sql.js]: Ah yes, removed!

File: src/lib/validations/sql.js (hunk 2 above)
Comment (pandah3) [reply src/lib/validations/sql.js]: Fixed!

File: src/lib/validations/sql.js (hunk 1 above)
Comment (pandah3) [reply src/lib/validations/sql.js]: It looks like we actually need this to pass down `isImport`. Reverted in #639

**Pull request merged**